presets/TARGET_MIMXRT1050_EVK/TARGET_BL_INTERNAL_FLASH/mbed_lib.json.template

```diff
@@ -0,0 +1,28 @@
+{{
+    "name": "mbed-bootloader-{target}-{preset}",
+    "target_overrides": {{
+        "*": {{
+            "target.macros_add": [
+                "MBED_CONF_STORAGE_STORAGE_TYPE=TDB_INTERNAL",
+                "MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS=(MBED_ROM_START + 64*1024 + 1984*2*1024 + 64*1024)",
```

Comment:
```suggestion
                "MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS=(MBED_ROM_START + 64*1024 + 1984*2*1024)",
```
? Is there double counting of the Bootloader ?

Reply:
@teetak01  indeed there is a gap between candidate storage and kvstore. But as we talked these optimization are not needed, we have here 8Mb memory of external flash that treaded as internal flash